test/index.js

```diff
@@ -3,15 +3,16 @@ var Test = require('segmentio-integration-tester');
 var helpers = require('./helpers');
```

Comment:
I probably could but I think given that we'll change this anyway -- as long as it works I'm okay with this being a massive giant for now.